dev-envs/linux/ssh.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -l
+IFS=$'\n\t'
+set -euxo pipefail
+
+# New versions require OpenSSL >= 1.1.1
+VERSION="V_9_3_P2"
+url="https://github.com/openssh/openssh-portable/archive/refs/tags/${VERSION}.tar.gz"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Why not installing it from apt?

Reply:
The base image is very old so it's easier to get the source directly.